Flick/Networking/NetworkManager.swift

```diff
@@ -340,5 +340,30 @@ class NetworkManager {
         }
     }
 
+    static func searchMedia(query: String, completion: @escaping ([Media]) -> Void) {
+        guard let searchBaseUrl = URL(string: "\(hostEndpoint)/api/search/") else { return }
+        var urlComp = URLComponents(url: searchBaseUrl, resolvingAgainstBaseURL: true)
+        urlComp?.queryItems = [
+            URLQueryItem(name: "is_movie", value: "true"),
+            URLQueryItem(name: "is_tv", value: "true"),
+            URLQueryItem(name: "query", value: query)
+        ]
+
+        guard let url = urlComp?.url?.absoluteString else { return }
+        AF.request(url, method: .get, headers: headers).validate().responseData { response in
```

Comment:
is the URLQueryItem building out the query URL? i woulda just done `\(hostEndpoint)/api/search?is_movie=true&is_tv=true&query=\(query)` lol but thats just personal preference :)